nrf_security/src/psa_crypto_driver_wrappers.c

```diff
@@ -2751,34 +2201,23 @@ psa_status_t psa_driver_wrapper_mac_compute(
             /* Key is stored in the slot in export representation, so
              * cycle through all known transparent accelerators */
 #if defined(PSA_CRYPTO_ACCELERATOR_DRIVER_PRESENT)
-#if defined(PSA_CRYPTO_DRIVER_TEST)
-            status = mbedtls_test_transparent_mac_compute(
-                attributes, key_buffer, key_buffer_size, alg,
-                input, input_length,
-                mac, mac_size, mac_length );
-            /* Declared with fallback == true */
-            if( status != PSA_ERROR_NOT_SUPPORTED )
-                return( status );
-#endif /* PSA_CRYPTO_DRIVER_TEST */
-#if defined(PSA_CRYPTO_DRIVER_CC3XX)
-
+#if defined(PSA_CRYPTO_DRIVER_HAS_MAC_SUPPORT_CC3XX)
             status = cc3xx_mac_compute(attributes, key_buffer, key_buffer_size, alg,
                 input, input_length,
                 mac, mac_size, mac_length);
             /* Declared with fallback == true */
             if( status != PSA_ERROR_NOT_SUPPORTED )
                 return( status );
-#endif /* PSA_CRYPTO_DRIVER_CC3XX */
+#endif /* PSA_CRYPTO_DRIVER_HAS_MAC_SUPPORT_CC3XX */
 // TODO: Either remove or adjust when mac is available in Oberon
```

Comment:
Not directly linked with this PR shouldn't a TODO always linked to a Jira ticket so it is findable

Reply:
Todos have been removed. Fixed!